.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.4
```

Comment:
2.3.8 is the latest. I prefer using latest one.
We are also thinking of using ruby 2.4 or 2.5

Reply:
Let’s set it to 2.3.8, this PR is to have a fixed version not to discuss what particular version. Or is it?

Reply:
### Issues
Enforcing a version for development is good so developers share the same version.

BUT though it works for development since ruby API / ABI compatibility https://www.ruby-lang.org/en/news/2013/12/21/ruby-version-policy-changes-with-2-1-0/ it will not be the case for the gems

Some gems (example the last version of LicenseFinder https://github.com/pivotal-legacy/LicenseFinder/blob/2a7f4ba50037b641523787da2b2d887046ccab0c/license_finder.gemspec#L10) will require higher version of Ruby. It will work on your development platform but not on production or on container without installing the new ruby.

### Upgrade to 2.4 or 2.5

2.4 and 2.5 @mikz  and @gsaslis  have done some branches already. Few failing tests like 10 or so so we probably want to upgrade later :)

Reply:
Ruby API compatibility is on MAJOR.MINOR. So you should lock only to `2.3`. The last one is a patch version and has no impact on the API.

Reply:
> 2.3.4 was based on what most are running currently. How important for you?

Not important it is just used for any package manager ... We can use Gemfile and use lazy version: 2.3

Reply:
In the INSTALL.md we say:
> The project supports **Ruby 2.3.x**.

So either we stick to `2.3.x` or we should change that comment too 😄